jobs/transforms/story_validation.py

```diff
@@ -75,3 +75,17 @@ def _get_change_domain_class(self, input_model): # pylint: disable=unused-argume
             return story_domain.StoryChange
         else:
             return None
+
+
+@validation_decorators.RelationshipsOf(story_models.StoryCommitLogEntryModel) # type: ignore[no-untyped-call, misc, name-defined]
+def story_commit_log_entry_model_relationships(model: Any) -> Iterator[
+    Tuple[Any, List[Type[story_models.StoryModel]]]]:
```

Comment:
```suggestion
def story_commit_log_entry_model_relationships(
		model: Any
) -> Iterator[Tuple[Any, List[Type[story_models.StoryModel]]]]:
```

Reply:
ditto elsewhere

also why is the model `Any`?

Reply:
done, changed the model type everywhere